packages/vaex-core/vaex/dataset.py

```diff
@@ -3585,6 +3585,33 @@ def add_virtual_columns_cartesian_velocities_to_polar(self, x="x", y="y", vx="vx
         if propagate_uncertainties:
             self.propagate_uncertainties([self[vr_out], self[vazimuth_out]])
 
+    def add_virtual_columns_polar_velocities_to_cartesian(self, x='x', y='y', azimuth=None, vr='vr', vazimuth='vazimuth', vx_out='vx', vy_out='vy', propagate_uncertainties=False):
```

Comment:
shell we keep the default arguments the same as add_virtual_columns_cartesian_velocities_to_polar, so vr_polar and vphi_polar?